WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2743,16 +2697,16 @@ title_guru = {
 	message = yes
 }
 title_genghis = {
-	dignity = 10
+	dignity = 500
```

Comment:
@loup99 

Genghis Khan was one of the greatest conquerers in human history. He created one of the largest empires the world had ever known. His title is the ultimate aspiration a steppe nomad can reach. He deserves that kind of Dignity.